Update active-directory-saas-promapp-tutorial.md

shaunrfield opened the pull request:
Updated to indicate the difference between 'required' and 'optional' mode in Promapp's single sign on implementation.  Also updated the testing steps to indicate that IdP authentication is not currently supported (this is a limited enforced by Azure presently).

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Blackmist commented:
This looks good, merge.

rmca14 commented:
Appears to be a merge conflict. Please resolve, thanks.

rmca14 commented:
I will resolve this merge conflict.

Blackmist commented:
Thanks Ross.

From: Ross McAllister [mailto:notifications@github.com]
Sent: Tuesday, March 31, 2015 7:48 PM
To: Azure/azure-content
Cc: Larry Franks
Subject: Re: [azure-content] Edit hdinsight-hadoop-linux-information.md (#3220)


I will resolve this merge conflict.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/3220#issuecomment-88286995>.

rmca14 commented:
Couldn't repro/induce merge conflict by checking out this article and running "git pull upstream master", so I created another branch, implemented the edits into the content, got the edits approved, and submitted a new pull request.

shaunrfield committed:
Update active-directory-saas-promapp-tutorial.md

Updated to indicate the difference between 'required' and 'optional' mode in Promapp's single sign on implementation.  Also updated the testing steps to indicate that IdP authentication is not currently supported (this is a limited enforced by Azure presently).

femila reviewed (commented):
@jeevansd Please review and approve
